package.json

```diff
@@ -9,29 +9,41 @@
     "test": "test"
   },
   "dependencies": {
+    "@ag-grid-community/client-side-row-model": "^31.3.2",
+    "@ag-grid-community/core": "^31.3.2",
+    "@eonasdan/tempus-dominus": "^6.7.7",
```

Comment:
I'm not sure about this library. The maintainer has stated they're "moving on to other projects". This suggests the library is essentially in maintenance mode and may not receive regular updates or bug fixes.

Reply:
Hmm, maybe we should just go with the built-in utilities?
https://getbootstrap.com/docs/5.2/content/reboot/#forms
Not very fancy, but probably pretty solid if we can figure out the formatting.

Reply:
gov.uk suggests 3 boxes:

https://design-system.service.gov.uk/components/date-input/

I'm not a huge fan of that, but it isn't terrible.